lightning/src/chain/onchaintx.rs

```diff
@@ -634,6 +643,26 @@ impl<ChannelSigner: Sign> OnchainTxHandler<ChannelSigner> {
 			}
 		}
 
+		self.process_onchain_events_awaiting_threshold_conf(cur_height, logger);
```

Comment:
Well, I wasn't 100% sure so I opted to include it in both methods, as it would have been called both times. Now removed it from `update_claims_view_from_requests` and inlined it into `update_claims_view_from_matched_txn` again.